tools/cli/commands-packages.js

```diff
@@ -1649,6 +1649,22 @@ main.registerCommand({
   } else {
     upgradePackageNames = options.args;
   }
+  var upgradePackagesWithoutCordova = _.reject(upgradePackageNames, function (name) {
+    return name.split(':')[0] === 'cordova'; 
+  });
```

Comment:
Yes, but I believe only existing code that is _actually being changed_ and new additions are being updated to ES6 at this time.